lib/input/check_headers.py

```diff
@@ -125,7 +125,7 @@ def check_validity(self, check_unavailable_uid='False', json_header_filename='he
 		return self.last_uid_read
 
 
-	def remove_unwanted_headers(self, to_remove=unwanted_uid, json_header_filename='headers.json'):
+	def remove_unwanted_headers(self, to_remove=unwanted_uid, in_json_header_filename="headers.json", out_json_header_filename="headers.json"):

```

Comment:
two variable names that differ only in prefix are confusing. Please use one variable with variable name as `json_headers`.